client/components/Navbar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const Navbar = () => (
+  <div>
+    <nav className="navbar is-light" aria-label="main navigation">
+      <div className="navbar-brand">
+        <a className="navbar-item" href="http://bulma.io">
+            Happy-Mate
+        </a>
+      </div>
+      <div className="navbar-end">
+        <a
```

Comment:
WTF....? Maybe a link that says "Happy-Mate" that goes to `/` would be better. ;)

Reply:
Sorry, prune this pointless link